build.gradle.kts

```diff
@@ -122,16 +123,16 @@ publishing {
             }
 
             developers {
-               developer {
```

Comment:
I would keep sksamuel here because most of the avro4k code originates from him.

Reply:
The developers are the maintainers, why putting him there? We still put the credits on the Readme, and for sure the code will evolve a lot, that will reduce more and more the original code 🤔